packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -734,12 +756,14 @@ class _ContextMenuRoute<T> extends PopupRoute<T> {
     CupertinoContextMenuBuilder? builder,
     super.filter,
     required Rect previousChildRect,
+    double scaleFactor = _kOpenScale,
```

Comment:
Is the default going to be used? Or the caller will always supply a scaleFactor value? Asking because this class seems to be private.

Reply:
I was a little scared to make it required, but since its private like you said, there's no need to.